datajoint_utilities/dj_worker/__init__.py

```diff
@@ -44,10 +45,10 @@ class WorkerLog(dj.Manual):
     _table_name = "~worker_log"
 
     @classmethod
-    def log_process_job(cls, process, worker_name="", db_prefix=""):
+    def log_process_job(cls, process, worker_name="", db_prefix=[""]):
```

Comment:
Generally,  a mutable default value is not a good idea. Can you make it a tuple?